interface/frontend/src/components/detect-and-select/CandidateList.vue

```diff
@@ -1,46 +1,158 @@
+    <template v-if="candidates.length">
+
+      <div class="row">
+        <div class="col-md-4">
+          <div id="accordion">
+            <template v-for="(candidate, index) in candidates">
+              <div class="card">
+                <div class="card-header cursor-pointer" @click="toggleShow(index)">
+                  <p class="mb-0">
+                    <b>Candidate {{ index + 1 }}</b> (p={{ candidate.probability_concerning }})
+                    <i class="pull-right" v-if="candidate._saving">Saving...</i>
+                  </p>
+                </div>
+
+                <div class="collapse" :class="{ show: selectedCandidateIndex == index }">
+                  <div class="card-block">
+                    <candidate :candidate="candidate" :index="index"></candidate>
+                    <a @click="dismiss(candidate)">
+                      <button type="button" class="btn btn-sm btn-secondary">Dismiss</button>
+                    </a>
+                    <a @click="mark(candidate)">
+                      <button type="button" class="btn btn-sm btn-danger">Mark concerning</button>
+                    </a>
+                  </div>
+                </div>
+              </div>
+
+            </template>
+          </div>
+        </div>
+        <div class="col-md-8">
+          <open-dicom :view="viewerData" :marker="marker"></open-dicom>
+        </div>
       </div>
-      </template>
-      <template v-else>
-        <p class="card-text">No candidates available.</p>
-      </template>
-    </div>
-    <div class="col-md-8"></div>
+    </template>
+    <template v-else>
+      <p class="card-text">No candidates available.</p>
+    </template>
+    <button @click="showDicom">Show DICOM</button>
   </div>
-</div>
 </template>
 
 <script>
-import Candidate from './Candidate'
+  import {EventBus} from '../../main.js'
+  import Candidate from './Candidate'
+  import OpenDicom from '../open-image/OpenDICOM'
 
-export default {
-  components: { Candidate },
-  data () {
-    return {
-      candidates: []
-    }
-  },
-  created () {
-    this.fetchCandidates()
-  },
-  methods: {
-    fetchCandidates () {
-      this.$axios.get('/api/candidates')
-        .then((response) => {
-          this.candidates = response.data
-        })
-        .catch(() => {
-          // TODO: error callback
-        })
+  export default {
+    components: {Candidate, OpenDicom},
+    data () {
+      return {
+        candidates: [],
+        selectedCandidateIndex: -1,
+        viewerData: {
+          type: 'DICOM',
+          prefixCS: ':/',
+          prefixUrl: '/api/images/metadata?dicom_location=/',
+          paths: []
+        }
+      }
+    },
+    computed: {
+      marker () {
+        if (!this.selectedCandidate) {
+          return null
+        }
+
+        var noduleCentroid = this.selectedCandidate.centroid
+        return this.selectedCandidate ? {x: noduleCentroid.x, y: noduleCentroid.y, z: noduleCentroid.z} : null
```

Comment:
This check is redundant since the condition has already been checked above. If anything, it should check for 
```
noduleCentroid ? { etc...} : null;
```
Also should prefer and lean toward `const` over `let` and `var`. `let` only when mutability is needed which in this case is not.
Also, since selectedCandidate.centroid has all the property needed for the return, I think the code can be simplified to just:

```
marker() {
return this.selectedCandidate 
    ? this.selectedCandidate.centroid || null 
    : null
}
```

Reply:
First comment - for sure, just a mistake. Second thing - will fix. Last one - I am creating new object by intention in order to keep candidate's original coordinates until user finished dragging and they are saved on server. What do you think?

Reply:
@Serhiy-Shekhovtsov Oh, that's a good point. If that's the case, you might want to do something like this:
```
if (!this.selectedCandidate || !this.selectedCandidate.centroid) return null

// Extract needed data and make sure they are immutable
const {x, y, z} = this.selectedCandidate.centroid

// Do stuff with data if needed
// ...

return {x,y,z}
```